app/frontend/components/shared/form/input-form-control.tsx

```diff
@@ -78,7 +78,7 @@ export const TextAreaFormControl = (props: IInputFormControlProps) => {
         {
           inputProps: { as: Textarea }, // Use Chakra's Textarea instead of Input
           validate: {
-            satisfiesLength: (str) => !props.required || (str?.length >= 1 && str?.length < 256) || "Input is invalid", // Use a default error message or use translation as needed
+            satisfiesLength: (str) => (str?.length >= 1 && str?.length < 256) || "Input is invalid", // Use a default error message or use translation as needed
```

Comment:
what happens if the value is an empty string and the property is not required? Expected behaviour is the input to be valid for blank value, but if there is a value present then it should validate it's length. Can you double check if this condition works?